auk-notebook.ipynb

```diff
@@ -645,7 +645,7 @@
    "source": [
     "## Overall Collection Characteristics\n",
     "\n",
-    "Change the variables in the following cell to manipulate the analysis you'll be running to understand overall collection characteristics. In this case, they mostly affect the visualization that we generate.\n"
+    "Change the variables in the following cell to manipulate the analysis you'll be running to understand overall collection characteristics. In this case, they mostly affect the visualization that we generate. You may wish to add or remove words to `STOP_WORDS_USER`in the user configuration cell to fine-tune the results.\n"
```

Comment:
maybe `fine-tune the results` -> `remove words that may be overwhelming your analysis`